web/secondary_toolbar.js

```diff
@@ -219,6 +219,7 @@ class SecondaryToolbar {
     // All items within the secondary toolbar.
     for (const { element, eventName, close, eventDetails } of buttons) {
       element.addEventListener("click", evt => {
+        eventBus.dispatch("secondarytoolbaraction", { source: this });
```

Comment:
If we're going to introduce a new event here (which I'm not crazy about personally), then at the very least it should be placed *after* the "reporttelemetry"-dispatch below.

Reply:
I'll do this and ask others later today how they prefer we do this.